src/show/components/Navbar/view.jsx

```diff
@@ -8,31 +8,29 @@ import { MdOutlineExplore } from 'react-icons/md';
 import { CgArrowsExpandRight } from 'react-icons/cg';
 import { BsArrowBarLeft, BsFillPersonLinesFill, BsPersonCircle } from 'react-icons/bs';
 
-import { mock } from 'process/helpers';
 import Paths from 'process/routes/paths';
+import { mock, onFieldChange } from 'process/helpers';
 
 import { Button, Modal, TextField } from '../';
 
-const Navbar = ({ currentUser, signIn, signOut, signUp }) => {
+const Navbar = ({ currentUser, signIn, signOut, signUp, transient, updateProps }) => {
   const navigate = useNavigate();
 
   const [openLogInModal, setOpenLogInModal] = useState(false);
   const [openLogOutModal, setOpenLogOutModal] = useState(false);
   const [openSignUpModal, setOpenSignUpModal] = useState(false);
 
-  const [payload, setPayload] = useState({});
-
-  const onFieldChange = (keyName, value) => setPayload({ ...payload, [keyName]: value });
+  const dipatchUpdateProps = payload => updateProps(payload);
 
   const submitSignIn = e => {
     e.preventDefault();
-    signIn(payload);
+    signIn(transient);
```

Comment:
I think after sigin in and signing up you'll have to reset the transient state back to the initial state, otherwise if you sign up and close the modal, and go to the sign in modal, email and password field are going to be already completed.

Reply:
done